neuralprophet/uncertainty.py

```diff
@@ -77,39 +78,21 @@ def predict(self, df: pd.DataFrame, df_cal: pd.DataFrame) -> pd.DataFrame:
             noncon_scores = self._get_nonconformity_scores(df_cal, step_number)
             q_hat = self._get_q_hat(df_cal, noncon_scores)
             y_hat_col = f"yhat{step_number}"
+            y_hat_lo_col = f"{y_hat_col} {min(self.quantiles) * 100}%"
+            y_hat_hi_col = f"{y_hat_col} {max(self.quantiles) * 100}%"
             if self.method == "naive" and self.symmetrical:
                 q_hat_sym = q_hat["q_hat_sym"]
-                q_hat_col = f"qhat{step_number}"
-                df[q_hat_col] = q_hat_sym
-                df[f"{y_hat_col} - {q_hat_col}"] = df[y_hat_col] - q_hat_sym
-                df[f"{y_hat_col} + {q_hat_col}"] = df[y_hat_col] + q_hat_sym
+                df[y_hat_lo_col] = df[y_hat_col] - q_hat_sym
+                df[y_hat_hi_col] = df[y_hat_col] + q_hat_sym
             elif self.method == "cqr" and self.symmetrical:
                 q_hat_sym = q_hat["q_hat_sym"]
-                q_hat_col = f"qhat{step_number}"
-                df[q_hat_col] = q_hat_sym
-                quantile_lo = str(min(self.quantiles) * 100)
-                quantile_hi = str(max(self.quantiles) * 100)
-                quantile_lo_col = f"{y_hat_col} {quantile_lo}%"
-                quantile_hi_col = f"{y_hat_col} {quantile_hi}%"
-                df[f"{quantile_lo_col} - {q_hat_col}"] = df[quantile_lo_col] - q_hat_sym
-                df[f"{quantile_lo_col} + {q_hat_col}"] = df[quantile_lo_col] + q_hat_sym
-                df[f"{quantile_hi_col} - {q_hat_col}"] = df[quantile_hi_col] - q_hat_sym
-                df[f"{quantile_hi_col} + {q_hat_col}"] = df[quantile_hi_col] + q_hat_sym
+                df[y_hat_lo_col] = df[y_hat_col] - q_hat_sym
```

Comment:
Needs to be:

`df[y_hat_lo_col] = df[y_hat_lo_col] - q_hat_sym`

As you are replacing the existing QR's lower quantile with that of CP. Also apply with this to upper quantile and assymetrical CQR.

Reply:
good catch, adressed in most recent commit